_topic_map.yml

```diff
@@ -238,6 +238,13 @@ Topics:
   - Name: AWS Service Broker
     File: aws_broker
 ---
+Name: Backup and restore
```

Comment:
This is my fault (because I forgot that 3.11 still uses sentence case) but this should be "Backup and Restore"

Reply:
I think I have it now, Andrea. Thanks!